README.md

```diff
@@ -200,7 +195,9 @@ The whitepaper is available through the project's [website](https://www.bosonpro
 
 We welcome contributions! Until now, Boson Protocol has been largely worked on by a small dedicated team. However, the ultimate goal is for all of the Boson Protocol repositories to be fully owned by the community and contributors. Issues, pull requests, suggestions, and any sort of involvement are more than welcome.
 
-If you have noticed a bug, [file an issue](/issues). If you have a large pull request, we recommend filing an issue first; small PRs are always welcome.
+If you have noticed a bug, please follow the [bug bounty procedure](https://github.com/bosonprotocol/community/blob/52725b04d1d3013dfc936d3d27ddc34019c6d02d/BugBountyProgram.md).
```

Comment:
@mischat  , currently bug bounties are security focused. If we keep this position for now, the. I'd modify the paragraph for security bugs only, and keep Github issues for all rest.

Reply:
I think every bug in the contracts should be within scope of the bug bounty, if the bug is trivial, then it would get the lowest level of payout; this is my position on this stuff. Let me know if you feel strongly about this.

Reply:
Removed reference to issues page. Will be included in next commit.